Telemetry/CorePing.swift

```diff
@@ -64,6 +69,13 @@ class CorePing: TelemetryPing {
 
         let timezoneOffset = NSTimeZone.localTimeZone().secondsFromGMT / 60
 
+        let usageCount = UsageTelemetry.getCount(prefs)
+        let usageTime = UsageTelemetry.getTime(prefs)
+        UsageTelemetry.reset(prefs)
```

Comment:
I assume `record` & `reset` can never be called concurrently (given that I don't see synchronization code)?